services/mariadb/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM maxexcloo/debian:latest
 MAINTAINER Max Schaefer <max@excloo.com>
-ENV MARIADB_USER docker \
-	MARIADB_PASS docker
```

Comment:
This syntax generated a `MARIADB_USER` var with `docker MARIADB_PASS docker` content. Found it doing a `echo $MARIADB_USER` while debug my mariadb container.